cloud-sql/mysql/activerecord/app/views/layouts/application.html.erb

```diff
@@ -4,11 +4,7 @@
     <title>Activerecord</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
```

Comment:
I don't understand why these lines needed to be deleted? Were they causing problems?

Reply:
>Were they causing problems?

Yes deploying to App Engine would error with:

> /usr/local/bundle/gems/execjs-2.8.1/lib/execjs/runtimes.rb:58:in `autodetect': Could not find a JavaScript runtime. See https://github.com/rails/execjs for a list of available runtimes. (ExecJS::RuntimeUnavailable)

Updating application.html.erb to match the Postgres sample resolved this error.